polkadot-test/Cargo.toml

```diff
@@ -0,0 +1,71 @@
+[package]
+name = "polkadot-test"
```

Comment:
Please rename to `polkadot-test-service` and move the folder also

Reply:
done in 886d76f8